maths/montecarlo.py

```diff
@@ -37,6 +37,29 @@ def circle(x: float, y: float):
     print("The total error is ", abs(pi - pi_estimate))
 
 
+def area_under_line_estimator(iterations: int):
```

Comment:
Given that we want this to be an algorithm that we could drop into a larger program, please streamline the area_under_line_estimator() function to return a floating point value.  Also, please move all calls to the __print()__ function outside of area_under_line_estimator().

Reply:
Thanks. I'll do that.